Krivorotova Tatyana/flex_layout/sass.scss

```diff
@@ -0,0 +1,129 @@
+@charset 'utf-8';
+
+body{
+    margin: 0;
+}
+
+%width-height-prcnt{
+    width: 100%;
+    height: 100%;
+}
+
+%flex-space-btw{
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+}
+
+@mixin min-height-width($height, $width){
+    min-height: $height;
+    min-width: $width;
+}
+
+.weather-widget{
+    @extend %width-height-prcnt;
+    margin: 0;
+    display: flex;
+    flex-wrap: wrap;
+    font-family: 'Lato', sans-serif;
+    color: #FFFFFF;
+}
+
+
+.weather-widget__city-card{
+    @extend %width-height-prcnt;
+    @include min-height-width(204px, 204px);
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.city-card-chicago{
+    background-color: #00C0E2;
+}
+
+.city-card-seattle{
+    background-color: #5AD998;
+}
+
+.city-card-brooklyn{
+    background-color: #7558F8;
+}
+
+.city-card-portland{
+    background-color: #E9C14D;
+}
+
+.city-card-schedule{
+    justify-content: space-between;
+    background-color: #E55679;
+}
+
+.city-card-sanjose{
+    background-color: #CB70D6;
+}
+
+.city-card-schedule__img{
+    margin-top: 25px;
+    margin-right: 10px;
+}
+
+.city-card-schedule__schedule-table{
+    @extend %width-height-prcnt;
+    @include min-height-width(99px, 204px);
+    background-color: #CB4C6C;
```

Comment:
if you use `scss` - it will be cool to store all colors in variables in separated file (some global color palette)